core/commands/root.go

```diff
@@ -190,6 +190,7 @@ var rootROSubcommands = map[string]*cmds.Command{
 	"dag": {
 		Subcommands: map[string]*cmds.Command{
 			"get":     dag.DagGetCmd,
+			"export":  dag.DagExportCmd,
```

Comment:
@Stebalien now that I know what this list is for - do we want to keep this? I.e. should someone be able to request a .car from the gateway ( it seems **really** cool, but maybe load implications... )

Reply:
Hm. Good point. Let's remove it for now and add a TODO to add it back later once we've done more load testing. It _would_ be cool, but this feature isn't going to get much hammering before the release.